src/Transformer/StringTransformer.php

```diff
@@ -5,22 +5,27 @@
 
 use Diaclone\Exception\MalformedInputException;
 use Diaclone\Exception\TransformException;
+use Diaclone\Resource\Item;
 use Diaclone\Resource\ResourceInterface;
-use Diaclone\Traits\EmojiTrait;
 use Exception;
 
 class StringTransformer extends AbstractTransformer
 {
-    use EmojiTrait;
+    protected $transformerClasses = [
```

Comment:
can we move this part to a `StringTransformer` inside the picr namespace in the app and keep this one inside diaclone just a simple string cast? because diaclone is more of a vendor library, and the name "string transformer" implies just a string cast instead of the additional emoji, html, and nonprintable transformers

Reply:
Yes, I think we can do this. Then we will need to refactor picr-api: create StringTransformer and replace namespace of all of using StringTransformer everywhere.

Reply:
@alexcoffee are you okay with this?

I would prefer this even though it might require a bit extra work